README.md

```diff
@@ -1,5 +1,7 @@
 # river
 
+[![Packaging status](https://repology.org/badge/vertical-allrepos/river.svg)](https://repology.org/project/river/versions)
```

Comment:
Maybe not as the first thing a newcomer sees? A dedicated "packaging status" section will be better IMO

Reply:
I feel like I always saw this first in repos using this, but I let @ifreund decide if/where he wants it

Reply:
I guess we could move it to the right side, see this [branch](https://github.com/novakne/river/tree/repology-badge-side)

Reply:
> I guess we could move it to the right side, see this [branch](https://github.com/novakne/river/tree/repology-badge-side)

That looks good to me, and doesn't waste as much space

Reply:
Mayby better add it next to building? 
![image](https://user-images.githubusercontent.com/71175207/115022635-8dc99680-9eb5-11eb-9ece-e9594c373d01.png)

Reply:
Having this aligned to the right next to building looks good to me.

Reply:
Done!